packages/core/android/gradle.properties

```diff
@@ -0,0 +1,5 @@
+Core_kotlinVersion=1.7.0
+Core_minSdkVersion=21
+Core_targetSdkVersion=31
+Core_compileSdkVersion=31
```

Comment:
Is it possible to upgrade `targetSdkVersion` and `compileSdkVersion` to `33` here ?

Reply:
Effectivement j'ai oublié cet endroit !